maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/util/FailsafeSummaryXmlUtils.java

```diff
@@ -84,12 +85,14 @@ public static RunResult toRunResult(File failsafeSummaryXml) throws Exception {
             String skipped = xpath.evaluate("/failsafe-summary/skipped", root);
             String failureMessage = xpath.evaluate("/failsafe-summary/failureMessage", root);
             String timeout = xpath.evaluate("/failsafe-summary/@timeout", root);
+            String flakes = xpath.evaluate("/failsafe-summary/flakes", root);
```

Comment:
nit: would be better to move this two lines up, right under `skipped`. `completed`, `errors`, `failures` and `skipped` all belong to the same type of field as `flakes`.